### PR TITLE
Feature #201 - Show real data tx-request and share-identity requests 

### LINKED ISSUE #201: Show real data tx-request and share-identity requests
**Description**
Right now tx-request and share-identity views use mocked data, once #199 is merged we can update those views with real data

- [ ] Update views with real data

### PR DESCRIPTION
**Pull request by albertandrejev**
**Description**
This PR will show real data from requests to appropriate views.
It will close #201 

**Screens**
Transaction request
tx-req-show
Transaction reject
tx-req-reject

Share identity
share-identity-main
Share identity reject
share-identity-reject

**Developer notes**
This PR was not fully completed because of lack of data in request object

### CODE REVIEW

**Comment by apanizo** [comment]
@albertandrejev I have read your note of "This PR is not completed because of lack of data in request object", but can you show (including screenshots) how they look like right now?

File: packages/sanes-chrome-extension/src/routes/share-identity/index.tsx [hunk 1]
```diff
@@ -31,8 +31,14 @@ const ShareIdentity = ({ location }: RouteComponentProps): JSX.Element => {
 
   return (
     <React.Fragment>
-      {action === 'show' && <ShowRequest onAcceptRequest={onAcceptRequest} showRejectView={showRejectView} />}
-      {action === 'reject' && <RejectRequest onBack={showRequestView} onRejectRequest={onRejectRequest} />}
+      {action === 'show' && (
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```
Comment (apanizo) [inline packages/sanes-chrome-extension/src/routes/share-identity/index.tsx]: For avoid the constant eslint disable check of `request!` I suggest to refactor code and do:
```
const request = requestContext.firstRequest;
const valid = validRequest(request, location, toast) <-- from checkRequest to validRequest
if (!valid) {
  return;
}
```

File: packages/sanes-chrome-extension/src/routes/tx-request/index.tsx [hunk 2]
```diff
@@ -31,8 +31,14 @@ const TxRequest = ({ location }: RouteComponentProps): JSX.Element => {
 
   return (
     <React.Fragment>
-      {action === 'show' && <ShowRequest onAcceptRequest={onAcceptRequest} showRejectView={showRejectView} />}
-      {action === 'reject' && <RejectRequest onBack={showRequestView} onRejectRequest={onRejectRequest} />}
+      {action === 'show' && (
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```
Comment (apanizo) [inline packages/sanes-chrome-extension/src/routes/tx-request/index.tsx]: Same here

File: packages/sanes-chrome-extension/src/routes/share-identity/index.tsx (hunk 1 above)
Comment (albertandrejev) [reply packages/sanes-chrome-extension/src/routes/share-identity/index.tsx]: Yes, that's was one of my first thoughts. But after this check I will be required to assign `request` variable to another non-undefined var.
like: 
```
// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
const requestValid = request!
```
And only after this I can use it in prop without any comment :(

**Pull request merged**